xlators/mgmt/glusterd/src/glusterd-volume-ops.c

```diff
@@ -2851,8 +2851,7 @@ glusterd_clearlocks_get_local_client_ports(glusterd_volinfo_t *volinfo,
             goto out;
         }
 
-        port = pmap_registry_search(THIS, brickname, GF_PMAP_PORT_BRICKSERVER,
-                                    _gf_false);
+        port = brickinfo->port;
```

Comment:
nit: I don't think you need the assignment at all - you use that info only once.